pkg/controller/networkpolicy/clustergroup_test.go

```diff
@@ -51,7 +51,7 @@ func TestProcessClusterGroup(t *testing.T) {
 			expectedGroup: &antreatypes.Group{
```

Comment:
is there a test for Service update (Service selector is patched)? If not, would it be easy to add one?

Reply:
Yes there is. L303.

Reply:
Sorry I meant a test where the K8s `Service` resource is updated (the label selector is updated) and the `Service` is referenced by a `ClusterGroup`. And not a test where the `ServiceReference` field is updated in a `ClusterGroup`.

Reply:
Thanks for the reminder. Added such testcase.